### PR TITLE
Add gauge chart to score component

### PR DESCRIPTION
**Pull request by neshang**
We decided to get rid of the analyse dialog in the end and re-design our progress container

In this PR:
1. some styling fixes overall
2. Add gauge chart to the score component

### CODE REVIEW

File: src/app/components/score-chart/score-chart.component.ts [hunk 1]
```diff
@@ -46,9 +48,80 @@ export class ScoreChartComponent implements OnChanges, OnInit {
...
+                    borderWidth: 0,
+                },
+            ],
+        },
+        series: [
+            {
+                type: "solidgauge",
+                radius: "62%",
+                innerRadius: "38%",
+                data: [20],
```
Comment (ansonmiu0214) [inline src/app/components/score-chart/score-chart.component.ts]: This should be `this.currentValue`

**Review by ansonmiu0214 — CHANGES_REQUESTED** [review]
Changes look good! Following from the Slack conversation, I'll commit the changes and fix this one as well.

**Review by ansonmiu0214 — COMMENTED** [review]
Looks great! One last thing - is the gauge fill color supposed to be "squared off" (as shown), or should it cover the circular region?
Screenshot 2021-04-02 at 9 14 56 PM

**Comment by neshang** [comment]
@ansonmiu0214 The solid gauge chart shouldn't be cropped out now. Thanks for spotting the bug!

I switched "@media screen and ( max-width: 1700px )" back to 1080px. 1700 seems to be too wide for my laptop..
With the sizes of the charts decreased, it looks good on my laptop now. Let me know if it still looks weird for you.

**Comment by ansonmiu0214** [comment]
@neshang 

I get this problem with a 1080px width (Chrome, 100% zoom) - is it different for you?

Screenshot 2021-04-03 at 1 56 01 PM

Chart update LGTM! I also added a y-axis maximum value to cap it at 100.

**Review by ansonmiu0214 — APPROVED** [review]
LGTM! :)

**Pull request merged**